api/clients/disperser_client.go

```diff
@@ -91,6 +103,14 @@ var _ DisperserClient = &disperserClient{}
 //	// Subsequent calls will use the existing connection
 //	status2, requestId2, err := client.DisperseBlob(ctx, otherData, otherQuorums)
 func NewDisperserClient(config *Config, signer core.BlobRequestSigner) *disperserClient {
+	if config == nil {
+		config = &Config{}
```

Comment:
should it return error if config is nil?

Reply:
yep good call: [f2100c8](https://github.com/Layr-Labs/eigenda/pull/849/commits/f2100c8f0055c27ac0678ae54e8f8c3eccc377cd)